src/lib/helpers/axeTester.ts

```diff
@@ -0,0 +1,11 @@
+import AxeBuilder from '@axe-core/playwright';
+import type { Page } from '@playwright/test';
+
+type ViolationsResult = Awaited<ReturnType<AxeBuilder['analyze']>>['violations'];
+
+export async function axeViolations(page: Page): Promise<ViolationsResult> {
```

Comment:
Amazing!

Reply:
Also, maybe put it outside of lib, so it isn't bundled unnecessarily.

Reply:
Would it bundle if its not imported by non-tests? But sure we can move it into a different folder. Any suggestion to which folder?

Reply:
Moved it to `/test-helpers` for now.

Reply:
It bundles everything that's inside lib, unless we specify otherwise in package.json `files`. We could eliminate helpers, but that's dangerous since we can create some non-test-related helpers in the future.